driver-legacy/src/main/com/mongodb/DBCursor.java

```diff
@@ -518,7 +518,9 @@ public DBCursor snapshot() {
      * @return a {@code DBObject} containing the explain output for this DBCursor's query
      * @throws MongoException if the operation failed
      * @mongodb.driver.manual reference/explain Explain Output
```

Comment:
Change the path to `reference/command/explain` to that it links to https://docs.mongodb.com/manual/reference/command/explain/ rather than https://docs.mongodb.com/manual/reference/explain.

Reply:
Done